umbral/keys.py

```diff
@@ -69,7 +69,7 @@ def __call__(self,
                 backend=default_backend()
             ).derive(password)
         except InternalError as e:
-            required_memory = 128 * 2**_scrypt_cost * 8 // (1024**1024)
+            required_memory = 128 * 2**_scrypt_cost * 8 // (10**6)
```

Comment:
```suggestion
required_memory = 128 * 2**_scrypt_cost * 8 // (1024**2)```

Reply:
This is more accurate. :)

Reply:
Or just remove it, I have a PR on cryptography.io cooking up lol

Reply:
Oh yeah! as soon as that's merged, this exception can be deprecated.

Reply:
Yeah, so maybe just undo this lol